PowerValidatedSolutions.psm1

```diff
@@ -32440,18 +32476,43 @@ Function Test-VsphereAuthentication {
 Export-ModuleMember -Function Test-VsphereAuthentication
 
 Function Test-SSOConnection {
+    <#
+        .SYNOPSIS
+        Check network connectivity to a vCenter Single Sign-On endpoint.
+
+        .DESCRIPTION
+        Checks the network connectivity to a vCenter Single Sign-On endpoint.
+        Supports testing a connection on port 443 (HTTPS).
+
+        .EXAMPLE
+        Test-SSOConnection -server sfo-m01-vcf01.sfo.rainpole.io
```

Comment:
```suggestion
        Test-SSOConnection -server sfo-m01-vc01.sfo.rainpole.io -port 443
```

Reply:
Addressed in 14b2e73bdb02f296b8ecbcda1032260e254946ce.